### PR TITLE
Add docs to install MM on ubuntu 1804

### PR DESCRIPTION
**Pull request by cpanato**
JIRA: https://mattermost.atlassian.net/browse/MM-11665

### CODE REVIEW

**Comment by cpanato** [comment]
@jasonblais Done, PTAL

File: source/install/install-ubuntu-1804-mysql.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**To install MySQL on Ubuntu Server 18.04:**
+
+1. Log into the server that will host the database, and open a terminal window.
+
+2. Install MySQL.
+
+  ``sudo apt install mysql-server``
+
+3. Run the ``mysql_secure_installation`` and follow the steps
```
Comment (jasonblais) [inline source/install/install-ubuntu-1804-mysql.rst]: Propose

```
Run ``mysql_secure_installation`` and follow the instructions. They are designed to set a few initial parameters that are more secure than the default Ubuntu (and MySQL) installation.
```

Comment (jasonblais) [reply source/install/install-ubuntu-1804-mysql.rst]: ^FYI - I updated the text. I had a second sentence included, but it's not necessary.

Comment (cpanato) [reply source/install/install-ubuntu-1804-mysql.rst]: done @jasonblais PTAL

Comment (jasonblais) [reply source/install/install-ubuntu-1804-mysql.rst]: @cpanato I see the original text still, perhaps you forgot to push?

Comment (cpanato) [reply source/install/install-ubuntu-1804-mysql.rst]: yep, I pushed to origin with is wrong :(
should be fixed now

**Pull request merged**